### ISSUE TITLE
Separate projects 

### ISSUE THREAD

**Issue by Wytamma** [python-friends/auto-scope#5]
I'm wondering how separate the image analysis and image capture portions of the project should be? Should auto-scope only contain the image acquisition portion of the project i.e. up to image stitching? We could have a sperate image analysis project (maybe a web app?) that has a GUI for image labelling and analysis.

**Comment by esteinig** [comment]
Yes, it depends on what we want to achieve with the overall project - is the point generating images only, or is it to provide the analysis for the automated slides as well? I suppose we could keep it cleaner by separating into two components, and thinking about it, there is little overlap, unless we want to do real-time analysis on the slide portions. 

So, yes, perhaps we should do the image analysis and GUI in a distinct project... any ideas for names?

**Comment by Wytamma** [comment]
I'm trying to envision how we will interact with the microscope. It might be useful if we have a GUI (or local web app) for driving the stage and previewing images, so maybe the analysis could integrate with that? Although, I can see how we might use the analysis system on images not captured with auto-scope as it should work independently from the capture system. If we want to do real-time analysis, we could always import the analysis module into auto-scope as required. 

Names...
- auto-analysis
- scope-pipe
- scope-analysis
- auto-scope-analysis

(:D I'm not committed to the '-')

**Comment by esteinig** [comment]
Yes, simple import into the capture system should be fine for real-time.  I agree the GUI should encompass the entire workflow. Can we call it  `scopy` - it's so cute!?

**Comment by Wytamma** [comment]
Scopy! 😂 I love it.
